servicenow/resources/resource_scripted_rest_api.go

```diff
@@ -0,0 +1,174 @@
+package resources
+
+import (
+	"github.com/coveooss/terraform-provider-servicenow/servicenow/client"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+const scriptedRestApiActive = "active"
+const scriptedRestApiConsumes = "consumes"
+const scriptedRestApiConsumes_customized = "consumes_customized"
```

Comment:
since it's static, your Resource does not need to "know" those values, remove them from the const and schema